lightkurve/targetpixelfile.py

```diff
@@ -1389,3 +1413,19 @@ def get_bkg_lightcurve(self, aperture_mask=None):
                               flux=np.nansum(self.flux_bkg[:, aperture_mask], axis=1),
                               flux_err=flux_bkg_err,
                               **keys)
+
+def open_fits(path):
```

Comment:
I think we can simply call it `open()`!  Can anyone think of a reason to call it `open_fits()`?

Reply:
I opted for `open_fits()` over `open()` to preserve the built-in python function `open()`. But if you think this won't be an issue I'm happy to simplify it!

In that case, it should probably throw an exception if `path` doesn't point to a fits file.

Reply:
I think I prefer `lightkurve.targetpixelfile.open()` for consistency with `astropy.io.fits.open()`.